### PR TITLE
Refactored writing dataframes to a file into utils

### PR DESCRIPTION
**Pull request by ramo-j**

### CODE REVIEW

File: dftimewolf/lib/utils.py [hunk 1]
```diff
@@ -52,6 +54,23 @@ def Compress(source_path: str, output_directory: Optional[str]=None) -> str:
   return filepath
 
 
+def WriteDataFrameToJsonl(df: pd.DataFrame) -> str:
+  """Writes a panda DataFrame to jsonl.
```
Comment (sydp) [inline dftimewolf/lib/utils.py:58]: pandas?

File: dftimewolf/lib/utils.py [hunk 2]
```diff
@@ -52,6 +54,23 @@ def Compress(source_path: str, output_directory: Optional[str]=None) -> str:
...
+
+  Args:
+    df: The frame to output.
```
Comment (sydp) [inline dftimewolf/lib/utils.py:61]: frame -> dataframe?

**Review by sydp — DISMISSED** [review]
lgtm w/ minor doc nit.

File: dftimewolf/lib/utils.py (hunk 1 above)
Comment (ramo-j) [reply dftimewolf/lib/utils.py]: Done.

File: dftimewolf/lib/utils.py (hunk 2 above)
Comment (ramo-j) [reply dftimewolf/lib/utils.py]: Done

**Review by sydp — APPROVED** [review]

**Pull request merged**